apps/dashboard/src/components/data-table/payment-product-table.tsx

```
@@ -102,7 +105,12 @@ function ActionsCell({ product }: { product: PaymentProduct }) {
         okButton={{
           label: "Delete",
           onClick: async () => {
-            await project.updateConfig({ [`payments.products.${product.id}`]: null });
+            const config = project.useConfig();
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)